go/cmd/dolt/commands/sql.go

```diff
@@ -418,107 +417,109 @@ func prepend(s string, ss []string) []string {
 }
 
 // Processes a single query and returns the new root value of the DB, or an error encountered.
```

Comment:
Your arch nemesis, an out of date comment!

Couple other methods need updating too, where you have changed the return type.